subsys/bluetooth/host/hci_core.c

```diff
@@ -1194,6 +1194,24 @@ static int hci_le_read_phy(struct bt_conn *conn)
 }
 #endif /* defined(CONFIG_BT_USER_PHY_UPDATE) */
 
+int bt_le_set_default_phy(uint8_t all_phys, uint8_t pref_tx_phy, uint8_t pref_rx_phy)
```

Comment:
I think it would be better YAGNI to just inline this `bt_le_set_default_phy` into `bt_conn_le_set_default_phy` and I would prefer that.

I understand this is the boilerplate for sending the specific command, and it even includes the not-yet-used argument `all_phys` (whatever that is), but I argue that this boilerplate is not completely general may not be always appropriate since it chooses `K_FOREVER`.

@jhedberg, do you agree with me?

Reply:
I approved now to not keep this PR over this. The benefit I see is just better readability.